monai/handlers/stats_handler.py

```diff
@@ -34,12 +34,26 @@ class StatsHandler:
     It can be used for any Ignite Engine(trainer, validator and evaluator).
     And it can support logging for epoch level and iteration level with pre-defined loggers.
 
+    Note that if `name` arg is None, will leverage `engine.logger` as default logger directly, otherwise,
+    inherit a new logger from `logging.RootLogger` with log level `INFO`. The default log level of `RootLogger`
+    is `WARNING`, so may need to call `logging.basicConfig(level=logging.INFO)` before running this handler
+    to enable the stats logging.
+
     Default behaviors:
         - When EPOCH_COMPLETED, logs ``engine.state.metrics`` using ``self.logger``.
         - When ITERATION_COMPLETED, logs
           ``self.output_transform(engine.state.output)`` using ``self.logger``.
 
-    Usage example is available in the tutorial:
+    Usage example::
+
+        logging.basicConfig(level=logging.INFO)
```

Comment:
For `basicConfig` it might be worth adding `stream=sys.stdout` to demonstrate output to screen which I think is a commonly desired behaviour.

Reply:
Sure, added in the latest commit.

Thanks.